opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/OpenTracingShimBuilder.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opentracingshim;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.api.trace.TracerProvider;
+import java.util.Objects;
+
+/** 
+ * Builder for creating an OpenTracing {@link io.opentracing.Tracer} that is implemented using the
+ * OpenTelemetry APIs.
+ */
+public class OpenTracingShimBuilder{
+    private Tracer tracer = getTracer(GlobalOpenTelemetry.getTracerProvider());
+    private OpenTracingPropagators propagators = OpenTracingPropagators.builder().build();
+
+    /** Returns a new {@code OpenTracingShimBuilder} instance. */
+    public static OpenTracingShimBuilder builder(){
```

Comment:
Our pattern for this is to have this method live in the `OpenTracingShim` class, rather than here.